src/fixate/config/local_config.json

```diff
@@ -1,3 +1,24 @@
 {
-    "INSTRUMENTS": {}
+  "INSTRUMENTS": {
+    "serial": {
+      "COM37": [
+        "address: 0,checksum: 28,command: 49,model: 6823,serial_number: 3697210019,software_version: 29440,start: 170,",
+        9600
+      ]
+    },
+    "visa": [
+      [
+        "RIGOL TECHNOLOGIES,DG1022 ,DG1D144904270,,00.03.00.09.00.02.11\n",
+        "USB0::0x09C4::0x0400::DG1D144904270::INSTR"
+      ],
+      [
+        "FLUKE,8846A,3821015,08/02/10-11:53\r\n",
+        "ASRL38::INSTR"
+      ],
+      [
+        "AGILENT TECHNOLOGIES,MSO-X 3014A,MY52160892,02.41.2015102200",
+        "USB0::0x0957::0x17A8::MY52160892::INSTR"
+      ]
+    ]
+  }
```

Comment:
This looks like local configuration. Should this have been committed?

Reply:
Not really, but it was convenient for testing. So I was lazy and left it there for now. Sooner or later I'm going to move the config out of the package directory to a more suitable location and will tidy up at that point.